github/repos.go

```diff
@@ -177,7 +177,7 @@ func (s *RepositoriesService) List(ctx context.Context, user string, opt *Reposi
 	}
 
 	// TODO: remove custom Accept headers when APIs fully launch.
-	acceptHeaders := []string{mediaTypeLicensesPreview, mediaTypeTopicsPreview}
+	acceptHeaders := []string{mediaTypeLicensesPreview, mediaTypeTopicsPreview, mediaTypeCodesOfConductPreview}
```

Comment:
`RepositoriesService.Get` method lists these in the following order:

```Go
acceptHeaders := []string{mediaTypeLicensesPreview, mediaTypeCodesOfConductPreview, mediaTypeTopicsPreview}
```

Let's update these to use the same, consistent order. Here, and all other lines in this PR.